tabular/src/autogluon/tabular/task/tabular_prediction/predictor.py

```diff
@@ -816,6 +817,120 @@ def fit_weighted_ensemble(self, base_models: list = None, name_suffix='_custom',
 
         return models
 
+    # TODO: v0.1 This can cause very strange issues related to index mismatches with original training data on extreme edge cases (multiclass where autogluon duplicated rows in data due to rare classes and eval_metric was log_loss)
+    #  This is a very rare case but needs to be fixed by v0.1 release
+    #  It might be good enough to do an inner join on training data, but it needs to be tested
+    def get_oof_pred(self, model: str = None, transformed=False) -> pd.Series:
+        """
+        Note: This is advanced functionality not intended for normal usage.
+
+        Returns the out-of-fold (OOF) training predictions of a model.
+        OOF predictions are unbiased estimates of model predictions on the training data.
```

Comment:
I think these sentences could be more clearly written as:
"Returns the out-of-fold (OOF) predictions for every row in the training data. OOF predictions may provide unbiased estimates of generalization accuracy (reflecting how predictions will behave on new data), as predictions for each row are only made using models that were fit to subset of data where this row was held-out"

Reply:
Added